types/bluebird-global/index.d.ts

```diff
@@ -65,7 +81,7 @@ declare global {
         each: typeof Bluebird.prototype.each;
         error: typeof Bluebird.prototype.error;
         filter: typeof Bluebird.prototype.filter;
-        finally: typeof Bluebird.prototype.finally;
+        // finally: typeof Bluebird.prototype.finally;
```

Comment:
Remove this comment.

Reply:
I have left it in intentionally, as to be in-line with `catch` and `then` (which, I assume, are hints that their definitions are being overridden below).